tests/integration.rs

```diff
@@ -311,6 +357,94 @@ mod tests {
         }
     }
 
+    impl DiskConfig for WindowsDiskConfig {
+        fn prepare_cloudinit(&self, _tmp_dir: &TempDir, _network: &GuestNetworkConfig) -> String {
+            String::new()
+        }
+
+        fn prepare_files(&mut self, tmp_dir: &TempDir, _network: &GuestNetworkConfig) {
+            let snapshot_cow_path =
+                String::from(tmp_dir.path().join("snapshot_cow").to_str().unwrap());
+
+            // truncate -s 1G /tmp/snapshot_cow
+            std::process::Command::new("truncate")
+                .args(&["-s", "1G"])
+                .arg(snapshot_cow_path.as_str())
+                .output()
+                .expect("Expect creating snapshot CoW image to succeed");
+
+            // losetup --find --show /tmp/snapshot_cow
+            let loopback_device = std::process::Command::new("losetup")
+                .arg("--find")
+                .arg("--show")
+                .arg(snapshot_cow_path.as_str())
+                .output()
+                .expect("Expect creating loopback device from snapshot CoW image to succeed");
+
+            self.loopback_device = String::from_utf8_lossy(&loopback_device.stdout)
+                .trim()
+                .to_string();
+
+            let random_extension = tmp_dir.path().file_name().unwrap();
+            let windows_snapshot_cow = format!(
+                "windows-snapshot-cow-{}",
+                random_extension.to_str().unwrap()
+            );
+
+            // dmsetup create windows-snapshot-cow-1 --table '0 2097152 linear /dev/loop1 0'
+            std::process::Command::new("dmsetup")
+                .arg("create")
+                .arg(windows_snapshot_cow.as_str())
+                .args(&[
+                    "--table",
+                    format!("0 2097152 linear {} 0", self.loopback_device).as_str(),
```

Comment:
What's this number...

Reply:
The number of sectors for 1GiB.

Reply:
Yes, in more general I was saying we shouldn't have hardcoded constants like this.

Reply:
Yep I fixed it :)